src/components/feature_activation/Features.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import ReactLoading from 'react-loading';
+import { Link } from 'react-router-dom';
+import { chunk, orderBy } from 'lodash';
+import { FEATURE_COUNT } from '../../constants';
+import FeatureRow from './FeatureRow';
+import colors from '../../index.scss';
+import PaginationURL from '../../utils/pagination';
+
+class Features extends React.Component {
+  constructor(props) {
```

Comment:
I think it would be good to add documentation to this component's `props`.
A suggestion would be to use the `propTypes` property of the component ( [reference](https://www.inkoop.io/blog/a-guide-to-js-docs-for-react-js/) ).

Reply:
Done in https://github.com/HathorNetwork/hathor-explorer/pull/242/commits/6ba68e2d1dbde262912f54faa028b3cdf743a981